### PR TITLE
swig 2.0.12 (new formula)

### PR DESCRIPTION
**Pull request by jbreams**
This adds a formula for the latest of the 2.x version of Swig.

### CODE REVIEW

**Comment by mistydemeo** [comment]
Is swig2 incompatible with swig 1.x? Is it packaged separately in other package managers?

**Comment by mistydemeo** [comment]
Sorry, ignore that comment. I missed that this is homebrew-versions.

**Comment by DomT4** [comment]
Please fixup comments ` brew audit --strict ` will have here. Thanks!

**Comment by jbreams** [comment]
Fixed.

File: swig2.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Swig2 < Formula
+  homepage "http://www.swig.org/"
+  url "https://downloads.sourceforge.net/project/swig/swig/swig-2.0.12/swig-2.0.12.tar.gz"
+  sha1 "4203c68f79012a2951f542018ff4358d838b5035"
```
Comment (DomT4) [inline swig2.rb]: You can kill the ` sha1 ` line. Only need one checksum, and ` sha256 ` is preferred :)

Comment (jbreams) [inline swig2.rb]: Sure.

**Comment by DomT4** [comment]
Just the nit over the double checksum and then this looks good.

**Comment by jbreams** [comment]
Fixed the nit.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/cf4f210fd30a19cf384f3fdec9384966fb5c769c. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**